app/main.py

```
@@ -146,6 +155,11 @@ def run(self):
         self.app.add_event_handler("startup", self.startup_event)
         return self.app
 
+    def run(self):
```

Ruff (0.13.3)

Redefinition of unused `run` from line 153

(F811)